src/ui/src/mainwindow.h

```diff
@@ -17,21 +18,29 @@ class MainWindow : public QMainWindow {
 	Q_OBJECT
  public:
 	explicit MainWindow(QWidget* parent = nullptr);
-	void LoadFile(const std::string& file_name);
+	void LoadFile(const std::string& trm_file, 
+								const std::string& t2d_file);
 	~MainWindow();
```

Comment:
As non-default dtor is defined, for rule of 5 (7) it's necessary to define or delete copy ctor, copy assignment operator, move ctor and move assignment operator.